tests/server/3-document.spec.js

```diff
@@ -71,7 +71,7 @@ describe('Document', () => {
           });
     });
 
-    it('should return err if something goes wrongtoken is used for accessing all documents', (done) => {
+    it('should return err if wrongtoken is used for accessing all documents', (done) => {
```

Comment:
### Critical

There should be space between wrong and token. It's concerning how much feedback has been and is still being delivered around `attention  to detail`.